NAMESPACE

```diff
@@ -431,6 +431,7 @@ export(vec_arith.difftime)
 export(vec_arith.logical)
 export(vec_arith.numeric)
 export(vec_arith_base)
+export(vec_as_names)
```

Comment:
Another possible name for this is `vector_make_names()`, to make the connection to `base::make.names()` more clear.

I'm not saying I like that better, just raising it. No matter what, it's a connection we should make in docs.

Reply:
I like `vec_as_names()` because it is similar to `vec_as_index()`. Good point about the connection, I have added a mention of `make.names()` in the doc.